pkg/utils/asynciter.go

```diff
@@ -49,7 +49,9 @@ type AsyncItemsCursor struct {
 	Cnt           int
 }
 
-func NewAsyncItemsCursor(container *v3io.Container, input *v3io.GetItemsInput, workers int) (*AsyncItemsCursor, error) {
+func NewAsyncItemsCursor(
+	container *v3io.Container, input *v3io.GetItemsInput,
+	workers int, shardingKeys []string) (*AsyncItemsCursor, error) {
```

Comment:
I believe that number of workers should be taken from the configuration and configuration should be read-only singleton  accessible from everywhere in the code of this library